configuration/helm-charts/ksqldb.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Integrating ksqlDB with Kafbat UI
+---
+
+# Integrating ksqlDB with Kafbat UI
+
+## Summary
+
+This document provides configuration details for integrating ksqlDB with Kafbat UI.
+
+Using Kafbat, you can easily integrate [ksqlDB](https://github.com/confluentinc/ksql) with Kafbat UI. This provides a powerful tool for visualizing and managing real-time streaming data.
+
+![ksqlDB architecture](../../.gitbook/assets/ksqldb-1.png)
+
+[ksqlDB](https://github.com/confluentinc/ksql) is a SQL engine based on Kafka Streams. ksqlDB provides a query layer that allows you to build event streaming applications on Kafka topics. Unlike Kafka Streams, ksqlDB allows you to create new streams or materialized views using SQL.
+
+## Configuration
+
+ksqlDB configuration is specified in `yamlApplicationConfig` section of `values.yaml` file.
```

Comment:
Hi, thanks for the PR. This part only applies to users running the app via helm charts, can we add some short info for other types of installations as well? Some might not understand how to apply this to their configs.

Reply:
@Haarolean Explicitly added that containers based on docker and docker-compose can also be configured using environment variables. ([48db4fa](https://github.com/kafbat/ui-docs/pull/26/commits/48db4faca4fb9923c0540a4df4699b1e356d5596))